command/unwrap.go

```diff
@@ -89,8 +89,10 @@ func (c *UnwrapCommand) Run(args []string) int {
 		return 2
 	}
 	if secret == nil {
-		c.UI.Error("Could not find wrapped response")
-		return 2
+		if Format(c.UI) == "table" {
+			c.UI.Info("Successfully unwrapped. There was no data in the wrapped token.")
```

Comment:
Rather than "wrapped token", I prefer "wrapped secret".  I know we use both terms on our website but at least in this file we use "wrapped secret" and I think that's a better way for users to think about wrapping.

Reply:
Done. Thanks!